cfme/fixtures/pytest_selenium.py

```diff
@@ -1237,14 +1240,29 @@ def none(self):
             return None
 
     @property
+    def is_patternfly(self):
+        return "selectpicker" in get_attribute(self._loc, "class")
+
+    @property
     def _el(self):
-        return move_to_element(self)
+        return element(self) if self.is_patternfly else move_to_element(self)
 
     @property
     def all_options(self):
         """Returns a list of tuples of all the options in the Select"""
-        els = execute_script("return arguments[0].options;", element(self))
-        return [self.Option(el.text, el.get_attribute('value')) for el in els]
+        # More reliable using javascript
+        script = dedent("""\
+            var result_arr = [];
+            var opt_elements = arguments[0].options;
+            for(var i = 0; i < opt_elements.length; i++){
+                var option = opt_elements[i];
+                result_arr.push([option.innerHTML, option.getAttribute("value")]);
+            }
+            return result_arr;
+        """)
+        options = execute_script(script, element(self))
+        parser = HTMLParser()
+        return [self.Option(parser.unescape(option[0]), option[1]) for option in options]
```

Comment:
Nope. Selenium apparently does that too (the `<None>` things and so, pure javascript yields `&lt;` and similar things). Using javascript, I can read the select always no matter in what position/visibility it is.